implentio/ui-toolkit
# Add Prettier and Eslint on Pre-commit

## Description
We need code readability (spacing, quotation mark differences and etc). We decided to use prettier and eslint along with lint-staged to this done.
We're introducing this after the repo has been worked on, so I ran it also agains the entire repo for now. We should now see prettier and eslint run automatically when you try to commit.

## Link to JIRA ticket
[Include the link here.](https://implentio.atlassian.net/browse/IUT-41)

## Applicable Screenshots
![Screenshot 2024-02-11 at 8 19 46 AM](https://github.com/implentio/ui-toolkit/assets/58754597/1a615771-0116-4d0e-8aa2-7b7de870e5ae)
Everything still working as intended in storybook

## Testing Instructions
Outline the steps required to test your changes effectively (Replace the following to what makes sense for your PR):
- Fetch and checkout my branch `git fetch && git checkout <branch-name>`
- add in a temporary file (copy any story from storybook and make a new filename and then make formatting wrong, like indentation). Then try to add and commit and it should make fixes to it.
- alternatively you can try running lint on specific file if you'd like as well.

## Checklist:
Check what is applicable by putting an 'x' in between the []:
